tmt/cli.py

```diff
@@ -309,13 +309,18 @@ def write_dl(
     '--force-color', is_flag=True, default=False,
     help='Forces tmt to use colors in the output and logging.'
     )
+@option(
+    '--pre-check', is_flag=True, default=False,
+    help='Run pre-checks on the git root. (Used by pre-commit wrapper).'
+    )
```

Comment:
This option seems to be a bit confusing as it is not relevat for regular usage of `tmt`. Also the argument reordering above sounds a bit hackish/complicated. We've had a quick chat with @lukaszachy and would like to rather propose introducing a dedicated `tmt` command which would cover the `pre-commit` use cases and would directly implement appropriate / adequate options.

As we don't want to move the `1.33` deadline any more, I'm proposing for `1.34`.

Reply:
The `hidden=True` seems to be a good compromise.  Could you please add it into this PR as a separate commit? Not sure if such one-liner is worth its own PR, having dedicated commit should be enough.

Reply:
Anything left to apply here, or can this conversation be resolved?